src/components/SecondaryPanes/index.js

```diff
@@ -121,10 +140,11 @@ const SecondaryPanes = React.createClass({
   },
 
   getStartItems() {
-    const scopesContent = this.props.horizontal ? this.getScopeItem() : null;
+    const scopesContent: any = this.props.horizontal ?
+      this.getScopeItem() : null;
     const isPaused = () => !!this.props.pauseData;
 
-    const items = [
+    const items: Array<StartItems> = [
       { header: L10N.getStr("breakpoints.header"),
         buttons: this.renderBreakpointsToggle(),
```

Comment:
Maybe this is what confuses me.  These are `StartItems` but there is a `buttons` attribute in the first one.